### DISCUSSION TITLE
Welcome to tldraw discussions!

### DISCUSSION THREAD

**Discussion by steveruizok** [tldraw/tldraw-v1#572] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To chat with the team and other tldraw users, also visit the [Discord channel](https://discord.gg/SBBEVCA4PG).

**Comment by ghost** [comment]
Hi all!
